src/components/Friends.js

```diff
@@ -6,35 +6,53 @@ class Friends extends React.Component {
     constructor() {
         super();
         this.state = {
-            friends: []
+            friends: [],
+            activeFriend: []
```

Comment:
Since `state` variables for `friends ` and `active` components have already been defined in this component's parent (`App.js`), there is no need to set up these variables again in `Friends.js`. You can simply pass `this.state.friends` from `App.js` as a prop for `<Friends />`.

_(this is also part of the solution for the issue you had with the component not immediately updating after a friend is deleted)_